AlphaWallet/Browser/ViewModel/BrowserHistoryCellViewModel.swift

```diff
@@ -39,7 +39,7 @@ struct BrowserHistoryCellViewModel {
     }
 
     var urlColor: UIColor? {
-        return Colors.appBackground
+        return UIColor(red: 0, green: 127, blue: 237)
```

Comment:
Great. Can you:

1. Do it for bookmarks too?
2. `return Colors.appTint` instead, which is the same color, which is basically R.color.azure(). Using `Colors.appTint` where it's the app tint color helps us switch the tint color. This is used in the other repo too and that app has a different tint color.

Reply:
I make changes into the Bookmark and History section. @hboon kindly review that.